### ISSUE TITLE
VSTest_ef087383-ee5e-42c7-9a53-ab56c98420f9\2.215.0\Modules\DTAExecutionHost.exe' failed with exit code 1

### ISSUE THREAD

**Issue by auro90** [microsoft/azure-pipelines-tasks#18031]
### Task name

VsTest - testAssemblies

### Task version

2.*

### Environment type (Please select at least one enviroment where you face this issue)

- [X] Self-Hosted
- [ ] Microsoft Hosted
- [ ] VMSS Pool
- [ ] Container

### Azure DevOps Server type

dev.azure.com (formerly visualstudio.com)

### Operation system

Windows 10

### Question

```shell
I have pipeline which was working few days back but now without any change it is throwing ##[error]No suitable test runtime provider was found for any source in this run.
 
I have latest microsoft .net in agent machine 
------------------------------------------------
2023-03-28T16:33:44.7197281Z ##[section]Starting: VsTest - testAssemblies
2023-03-28T16:33:44.7327432Z ==============================================================================
2023-03-28T16:33:44.7327854Z Task         : Visual Studio Test
2023-03-28T16:33:44.7328430Z Description  : Run unit and functional tests (Selenium, Appium, Coded UI test, etc.) using the Visual Studio Test (VsTest) runner. Test frameworks that have a Visual Studio test adapter such as MsTest, xUnit, NUnit, Chutzpah (for JavaScript tests using QUnit, Mocha and Jasmine), etc. can be run. Tests can be distributed on multiple agents using this task (version 2).
2023-03-28T16:33:44.7328993Z Version      : 2.215.0
2023-03-28T16:33:44.7329139Z Author       : Microsoft Corporation
2023-03-28T16:33:44.7329355Z Help         : https://docs.microsoft.com/azure/devops/pipelines/tasks/test/vstest
2023-03-28T16:33:44.7329621Z ==============================================================================
2023-03-28T16:33:45.9605727Z Running tests using vstest.console.exe runner.
2023-03-28T16:33:45.9606436Z ======================================================
2023-03-28T16:33:45.9607248Z Test selector : Test assemblies
2023-03-28T16:33:45.9607827Z Test filter criteria : TestCategory=smoke
2023-03-28T16:33:45.9608332Z Search folder : C:\AzureAgent\_work\r1\a
2023-03-28T16:33:45.9695735Z Action when minimum tests threshold not met : donothing
2023-03-28T16:33:45.9696029Z Minimum tests expected to be run: 0
2023-03-28T16:33:46.1031885Z Run in parallel : false
2023-03-28T16:33:46.1032419Z Run in isolation : false
2023-03-28T16:33:46.1032893Z Path to custom adapters : undefined
2023-03-28T16:33:46.1034138Z Other console options : undefined
2023-03-28T16:33:46.1034611Z Code coverage enabled : false
2023-03-28T16:33:46.1035322Z Diagnostics enabled : false
2023-03-28T16:33:46.1041917Z Run the tests locally using vstest.console.exe
2023-03-28T16:33:46.1042191Z ========================================================
2023-03-28T16:33:46.1043004Z Source filter: **\*test*.dll,!**\*TestAdapter.dll,!**\obj\**
2023-03-28T16:33:46.4299696Z ##[warning]Test Impact (Run only Impacted tests) and Code Coverage data collector will not work.
2023-03-28T16:33:46.4358734Z ##[warning]Test Impact (Run only Impacted tests) and Code Coverage data collector will not work.
2023-03-28T16:33:46.5479820Z ##[warning]Test Impact (Run only Impacted tests) and Code Coverage data collector will not work.
2023-03-28T16:33:46.5481823Z [command]C:\AzureAgent\_work\_tasks\VSTest_ef087383-ee5e-42c7-9a53-ab56c98420f9\2.215.0\Modules\DTAExecutionHost.exe --inputFile C:\AzureAgent\_work\_temp\input_4f812990-cd86-11ed-8733-7f08cfe06d81.json
2023-03-28T16:33:46.5506827Z ======================================================
2023-03-28T16:33:46.9140913Z ##########################################################################
2023-03-28T16:33:46.9142834Z DtaExecutionHost version 19.210.32906.4.
2023-03-28T16:33:47.2454972Z Starting TestExecution Model...
2023-03-28T16:33:49.1698490Z Result Attachments will be stored in LogStore
2023-03-28T16:33:49.3211380Z Run Attachments will be stored in LogStore
2023-03-28T16:33:49.3277052Z Result Attachments will be stored in LogStore
2023-03-28T16:33:49.4883472Z Result Attachments will be stored in LogStore
2023-03-28T16:33:49.4883817Z Run Attachments will be stored in LogStore
2023-03-28T16:33:49.5079133Z Updated Run Settings:
2023-03-28T16:33:49.5087083Z <RunSettings>
2023-03-28T16:33:49.5087310Z   <RunConfiguration>
2023-03-28T16:33:49.5087533Z     <BatchSize>1000</BatchSize>
2023-03-28T16:33:49.5087802Z     <ResultsDirectory>C:\AzureAgent\_work\_temp\TestResults</ResultsDirectory>
2023-03-28T16:33:49.5088073Z   </RunConfiguration>
2023-03-28T16:33:49.5088248Z </RunSettings>
2023-03-28T16:33:49.5164844Z **************** Starting test execution *********************
2023-03-28T16:33:49.5244359Z [command]C:\AzureAgent\_work\_tool\VsTest\17.6.0-preview-20230223-05\x64\tools\net462\Common7\IDE\Extensions\TestPlatform\vstest.console.exe "@C:\AzureAgent\_work\_temp\n5dqf5ho0qe.tmp"
2023-03-28T16:33:49.9202228Z Microsoft (R) Test Execution Command Line Tool Version 17.6.0-preview-20230223-05 (x64)
2023-03-28T16:33:49.9202931Z Copyright (c) Microsoft Corporation.  All rights reserved.
2023-03-28T16:33:49.9255043Z vstest.console.exe "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\E3DProductTestAutomation.dll"
2023-03-28T16:33:49.9255887Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.AdapterUtilities.dll"
2023-03-28T16:33:49.9256718Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.CommunicationUtilities.dll"
2023-03-28T16:33:49.9257355Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.CoreUtilities.dll"
2023-03-28T16:33:49.9257790Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.CrossPlatEngine.dll"
2023-03-28T16:33:49.9258224Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.PlatformAbstractions.dll"
2023-03-28T16:33:49.9258664Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.Utilities.dll"
2023-03-28T16:33:49.9259100Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.Common.dll"
2023-03-28T16:33:49.9259584Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.dll"
2023-03-28T16:33:49.9260113Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.dll"
2023-03-28T16:33:49.9260595Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.ObjectModel.dll"
2023-03-28T16:33:49.9261071Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.TestFramework.Extensions.dll"
2023-03-28T16:33:49.9261545Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.TestFramework.dll"
2023-03-28T16:33:49.9261993Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\TechTalk.SpecFlow.MSTest.SpecFlowPlugin.dll"
2023-03-28T16:33:49.9262457Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\cs\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9262926Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\cs\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9263390Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\cs\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9263857Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\cs\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9264343Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\cs\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9264830Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\de\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9265298Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\de\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9265764Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\de\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9266231Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\de\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9266717Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\de\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9267398Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\es\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9267874Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\es\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9269047Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\es\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9269641Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\es\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9270130Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\es\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9270615Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\fr\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9271087Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\fr\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9271545Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\fr\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9272014Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\fr\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9272501Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\fr\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9272988Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\it\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9273466Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\it\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9273924Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\it\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9274398Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\it\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9274876Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\it\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9275361Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ja\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9275835Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ja\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9276292Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ja\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9276767Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ja\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9277246Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ja\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9277730Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ko\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9278204Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ko\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9278666Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ko\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9279140Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ko\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9279621Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ko\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9280112Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pl\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9280579Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pl\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9281035Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pl\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9281590Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pl\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9282536Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pl\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9283035Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pt-BR\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9283510Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pt-BR\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9284020Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pt-BR\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9284495Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pt-BR\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9284983Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\pt-BR\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9285485Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ru\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9285952Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ru\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9286418Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ru\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9286883Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ru\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9287704Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\ru\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9288120Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\testhost.dll"
2023-03-28T16:33:49.9288538Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\tr\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9289011Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\tr\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9289472Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\tr\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9289944Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\tr\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9290423Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\tr\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9290920Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hans\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9291401Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hans\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9291879Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hans\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9292363Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hans\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9292856Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hans\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9293358Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hant\Microsoft.TestPlatform.CommunicationUtilities.resources.dll"
2023-03-28T16:33:49.9293840Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hant\Microsoft.TestPlatform.CoreUtilities.resources.dll"
2023-03-28T16:33:49.9294320Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hant\Microsoft.TestPlatform.CrossPlatEngine.resources.dll"
2023-03-28T16:33:49.9294918Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hant\Microsoft.VisualStudio.TestPlatform.Common.resources.dll"
2023-03-28T16:33:49.9295419Z "C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\zh-Hant\Microsoft.VisualStudio.TestPlatform.ObjectModel.resources.dll"
2023-03-28T16:33:49.9296196Z /TestCaseFilter:"TestCategory=smoke"
2023-03-28T16:33:49.9296471Z /Settings:"C:\AzureAgent\_work\_temp\orbnqgrxffh.tmp.runsettings"
2023-03-28T16:33:49.9296711Z /Logger:"trx"
2023-03-28T16:33:49.9296912Z /TestAdapterPath:"C:\AzureAgent\_work\r1\a"
2023-03-28T16:33:50.5559981Z Starting test execution, please wait...
2023-03-28T16:33:50.7195222Z No suitable test runtime provider was found for any source in this run.
2023-03-28T16:33:50.7196058Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.AdapterUtilities.dll (.NETStandard,Version=v1.0, X64)
2023-03-28T16:33:50.7196755Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.CommunicationUtilities.dll (.NETStandard,Version=v2.0, X64)
2023-03-28T16:33:50.7197344Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.CoreUtilities.dll (.NETStandard,Version=v2.0, X64)
2023-03-28T16:33:50.7197923Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.CrossPlatEngine.dll (.NETStandard,Version=v2.0, X64)
2023-03-28T16:33:50.7198478Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.TestPlatform.Utilities.dll (.NETStandard,Version=v2.0, X64)
2023-03-28T16:33:50.7199045Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.Common.dll (.NETStandard,Version=v2.0, X64)
2023-03-28T16:33:50.7199615Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\TechTalk.SpecFlow.MSTest.SpecFlowPlugin.dll (.NETStandard,Version=v2.0, X64)
2023-03-28T16:33:50.7200263Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.Interface.dll (.NETPortable,Version=v4.5,Profile=Profile259, X64)
2023-03-28T16:33:50.7200988Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.TestFramework.Extensions.dll (.NETPortable,Version=v4.5,Profile=Profile259, X64)
2023-03-28T16:33:50.7201662Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.TestFramework.dll (.NETPortable,Version=v4.5,Profile=Profile259, X64)
2023-03-28T16:33:50.7202306Z Skipping source: C:\AzureAgent\_work\r1\a\_E3D-ProductTest-Marine-SmokeTest\drop\Microsoft.VisualStudio.TestPlatform.MSTestAdapter.PlatformServices.dll (.NETStandard,Version=v1.5, X64)
2023-03-28T16:33:50.7324053Z A total of 80 test files matched the specified pattern.
2023-03-28T16:33:50.7569197Z ##[error]No suitable test runtime provider was found for any source in this run.
2023-03-28T16:33:50.7645001Z Vstest.console.exe exited with code 1.
2023-03-28T16:33:50.7645350Z **************** Completed test execution *********************
2023-03-28T16:33:50.7698773Z Test results files: 
2023-03-28T16:33:50.9908878Z Completed TestExecution Model...
2023-03-28T16:33:51.0018728Z ##[warning]Vstest failed with error. Check logs for failures. There might be failed tests.
2023-03-28T16:33:51.0021574Z ##[error]Error: The process 'C:\AzureAgent\_work\_tasks\VSTest_ef087383-ee5e-42c7-9a53-ab56c98420f9\2.215.0\Modules\DTAExecutionHost.exe' failed with exit code 1
2023-03-28T16:33:51.0025759Z ##[error]Vstest failed with error. Check logs for failures. There might be failed tests.
2023-03-28T16:33:51.0096225Z ##[section]Finishing: VsTest - testAssemblies
```

**Comment by ManikandanMani** [comment]
I too face the sane issue issue when test ran using vstest version  2.220.0 on azure agents version 3.220.1. The agent is a self hosted service agent.
`Error: The process: VSTest_ef087383-ee5e-42c7-9a53-ab56c98420f9\2.220.0\Modules\DTAExecutionHost.exe' failed with exit code 1`

**Comment by jeremytbrun** [comment]
Looking for a fix as well.

**Comment by janhagenaar** [comment]
We are having the same issue.

Temporarily fix is to downgrade to 17.5.0.

Code snippet in Azure Pipelines:
`- task: VisualStudioTestPlatformInstaller@1
       displayName: Install VS Test
       inputs:
         packageFeedSelector: 'nugetOrg'
         versionSelector: 'specificVersion'
         testPlatformVersion: '17.5.0' # 17.6.0 seems to have a bug, which causes tests to fail with error 'no suitable testprovider found'`

**Comment by mbrankintrintech** [comment]
Same issue, Microsoft-hosted
